tests/testsuite/kubevirtresource.go

```diff
@@ -188,7 +188,7 @@ func UpdateKubeVirtConfigValue(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 	Expect(err).ToNot(HaveOccurred())
 
 	if equality.Semantic.DeepEqual(kv.Spec.Configuration, kvConfig) {
-		return kv
+		return kv, false
```

Comment:
The `Expect(CurrentSpecReport().IsSerial).To(BeTrue(), "Tests which alter the global kubevirt configuration must not be executed in parallel, see https://onsi.github.io/ginkgo/#serial-specs")` should be moved to top, imho